### ISSUE TITLE
The user profile button is announced as a link by a screen reader.  

### ISSUE THREAD

**Issue by McManning** [osuresearch/auth#2]
The user profile button is announced as a link by a screen reader.  
•	[WCAG 1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)
•	Recommendation: It’s always best to use native HTML <a> for elements that navigates to new content; use <button> for activating functionality. However, if editing the element type cannot happen, change an anchor to a button with role=”button”.
•	Priority: Moderate
